pkg/datapath/linux/ipsec.go

```diff
@@ -214,24 +214,24 @@ func (n *linuxNodeHandler) enableIPsecIPv4(newNode *nodeTypes.Node, nodeID uint1
 			}
 
 			for _, cidr := range n.nodeConfig.IPv4PodSubnets {
-				spi, err = ipsec.UpsertIPsecEndpoint(wildcardCIDR, cidr, localIP, remoteIP, nodeID, newNode.BootID, ipsec.IPSecDirOut, zeroMark, updateExisting)
+				spi, err = ipsec.UpsertIPsecEndpoint(wildcardCIDR, cidr, localIP, remoteIP, nodeID, newNode.BootID, ipsec.IPSecDirOut, zeroMark, updateExisting, ipsec.DefaultReqID)
```

Comment:
I'm a little bit confused as to how these ipsec changes are related to the commits purpose of skipping un-needed endpoint logs.

Reply:
@tommyp1ckles So you understand the need for this as a whole, to remove only EncryptedOverlay XFRM policies when this feature is disabled, and IPSec feature as hole remains? And rather, you think the commit in which the change to plumb the ReqID (the means to do the former) is the wrong place for it?

Reply:
Hmm, [this commit](https://github.com/cilium/cilium/pull/31757/commits/cb14fdba63432da61cdcd42ce80ba4cdd63b60b6) looks bit out of place. @ldelossa  is it possible something got squashed incorrectly and ipsec req id change got fused with endpoint log commit ?

Reply:
Ah! I didn't even realize that. Yes this looks odd. I imagine we can drop this as long as the other reqID diff gets in there appropriately. 

Yeah. probably rebase nonsense.